pss/pss.go

```diff
@@ -29,6 +29,13 @@ type Pss struct {
 	localStore chunk.Store
 }
 
+// Monitor is used for tracking status changes in sent messages
```

Comment:
i know these chunks are trojan, but let's be more explicit than `messages` here to avoid confusion

Reply:
fixed, made it explicit